docs/docs/user/getting-started/components.md

```diff
@@ -1,16 +1,14 @@
-# Working with KPOps
-
-## Components
+# Components
 
 This section explains the different components of KPOps, their usage and configuration.
 
-### KubernetesApp
+## KubernetesApp
 
-#### Usage
+### Usage
 
-Can be used to deploy any app in Kubernetes using helm. An example is a Redis database.
+Can be used to deploy any app in Kubernetes using helm, for example, a REST Service that serves Kafka Data.
```

Comment:
please keep capitalization of proper nouns :)

```suggestion
Can be used to deploy any app using Helm in Kubernetes, such as a REST Service that serves Kafka data.
```

Reply:
done